package io.task.api.app.utils;

public class UserNotFoundException extends RuntimeException{

}
